include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -12711,6 +12711,15 @@
           "capabilities" : [ "VectorComputeINTEL" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "FPMaxErrorDecorationINTEL",
+          "value" : 6170,
+          "parameters" : [
+            { "kind" : "LiteralInteger", "name" : "'Max Error'" }
```

Comment:
The spec says it's float:
`_Max Error_ is a positive 32-bit float type number representing the maximum acceptable relative error.`

Reply:
I do not see an equivalent for floating point in the JSON file. Currently, any floating point is represented using a 32-bit integer that is the 32-bit representation of the floating point number.
Not sure what is the correct entry here.

Reply:
Based on some discussions, I changed this 'kind' to 'LiteralFloat'.

Thanks